src/app.cpp

```diff
@@ -167,6 +172,7 @@ int main(int argc, char* argv[]) {
 	args::Positional<std::string> inarg(parser, "infile", "Input File", args::Options::Required);
 	args::Positional<std::string> outarg(parser, "outfile", "Output File", args::Options::Required);
 	args::Flag tecmp_only(parser, "tecmp-only", "Only process TECMP packets, drop others", { "tecmp-only" }, args::Options::Single);
+	args::Flag ignore_replay_data(parser, "ignore-replay-data", "Ignore message type replay data", { "ignore-replay-data" }, args::Options::Single);
```

Comment:
"ignore" means do nothing with them (pass them as is), here you are "dropping" them.
```suggestion
	args::Flag drop_replay_data(parser, "drop-replay-data", "Drop replay data messages", { "drop-replay-data" }, args::Options::Single);
```